jsk_arc2017_baxter/launch/setup/include/stereo_astra_hand.launch

```diff
@@ -56,10 +54,24 @@
       </include>
       <node name="left_rgb_static_tf_publisher"
             pkg="tf" type="static_transform_publisher"
-            args="0.04 0.04 0 0 0 0 left_hand_camera_left_depth_optical_frame left_hand_camera_left_rgb_optical_frame 20" />
+            args="0.030 0.018 -0.000 -0.000 0.000 -0.001 left_hand_camera_left_depth_optical_frame left_hand_camera_left_rgb_optical_frame 20" />
       <node name="left_depth_static_tf_publisher"
             pkg="tf" type="static_transform_publisher"
-            args="$(arg left_hand_stereo_left_transform) left_hand left_hand_camera_left_depth_optical_frame 20" />
+            args="-0.020 -0.056 -0.005 1.550 0.029 -0.003 left_hand left_hand_camera_left_depth_optical_frame 20" />
+      <!-- XXX
+        <node name="marker_6dof"
+              pkg="jsk_interactive_marker" type="marker_6dof">
+          <rosparam>
+            object_type: cube
+            frame_id: left_hand
+            publish_tf: true
+            tf_frame: left_hand_camera_left_depth_optical_frame
+            object_x: 0.1
+            object_y: 0.1
+            object_z: 0.1
```

Comment:
actually, it seems `marker_6dof` has bug in the initialization of the pose from rosparam.
so I removed the initialization and it is set to 0 0 0 0 0 0.